evap/student/templates/student_vote.html

```diff
@@ -29,8 +29,9 @@
         </div>
     </small>
     {% endif %}
+    <div id="restore-info" class="bs-callout bs-callout-info" style="display: none">{% trans "Your answers have been restored for you. You can continue answering questions and then submit the questionnaire." %}</div>
```

Comment:
does this help? i'd say this PR is a bugfix from a UX point of view and this says "oh yeah that bug really doesn't happen anymore"